src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf) format("book"),
+    url(./fonts/Avenir-Heavy.ttf) format("heavy");
+}
+
+body {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  font-weight: 300;
+}
```

Comment:
move fonts to html and inherit where its needed

for now font doesnt work for input
![Screenshot 2022-10-28 at 10 53 00](https://user-images.githubusercontent.com/99331793/198534739-84841638-2ef8-41a3-9f53-a8e287e89623.png)